sig-pm/README.md

```diff
@@ -81,6 +81,7 @@ Chairs:
 
 	-	Jaice Singer DuMars (Google); Area: Program Management & Release
 	-	Dustin Kirkland (Google); Area: Product Management
+    -   Stephen Augustus (Red Hat); Area: Product Management
```

Comment:
Formatting nit, but otherwise LGTM. Thank you Stephen for all of the work you're doing.